compiler/luci/env/include/luci/UserSettings.h

```diff
@@ -33,6 +33,7 @@ struct UserSettings
     MuteWarnings,
     DisableValidation,
     ProfilingDataGen,
+    MemoryPlanGen,
```

Comment:
I have some concerns about "Memory" in this name:
We plan memory AND execution order at the same time...

So we can name it "ExecutionPlanGen", but this sounds too generic for me...

Reply:
Maybe smth like:
`InterpreterPlanGen` ?